### DISCUSSION TITLE
How to preserve carriage return at the end of a line?

### DISCUSSION THREAD

**Discussion by Truckle** [datenstrom/community#114] [Ask a question] [answered]
Yellow ignores SINGLE carriage returns and thus runs end-of-lines together with the next line. There are many instances when composing a markdown document when I want a certain block of text to have only SINGLE carriage returns instead of being DOUBLE spaced. I realize Markdown automatically sees two line feeds as a new paragraph. I need to do single line feeds without numbering and bullet points and have it stay that way when yellow renders the markdown to HTML.

How might I adjust the engine to render a single HTML line break or <br /> for single carriage returns in the content text file? I'm not a php whiz, so please be patient ;)

**Comment by markseuffert** [comment, marked as answer]
Here's how to insert a line break in [Markdown](https://en.wikipedia.org/wiki/Markdown). Leave 2 spaces at the end of a line to create a line break. Alternatively use HTML and insert `<br />` where you want to have the break. 

Hope this helps.

**Comment by Truckle** [comment]
Got it. Thanks.
